cli/admin/src/main/java/io/pravega/cli/admin/dataRecovery/RecoverFromStorageCommand.java

```diff
@@ -542,7 +537,10 @@ private void flushToStorage(DebugStreamSegmentContainer debugSegmentContainer) {
     }
 
     public static CommandDescriptor descriptor() {
-        return new CommandDescriptor(COMPONENT, "recover-from-storage", "Recover the state of a container from what is present on tier-2.");
+        return new CommandDescriptor(COMPONENT, "recover-from-storage", "Recover the state of a container from what is present on tier-2.",
+                new ArgDescriptor("start-container-id", "The start container Id of the Segment Container that needs to be recovered, " +
+                "if given as \"all\" all the containers will be recovered. If given as container id without "), new ArgDescriptor("end-container-id", "The end container Id of the Segment Container that needs to be recovered, " +
```

Comment:
If given as container id without? is the sentence complete?

Reply:
fixed it.